model-test/src/main/java/org/jboss/as/model/test/ModelTestModelControllerService.java

```diff
@@ -471,6 +473,9 @@ protected boolean boot(List<ModelNode> bootOperations, boolean rollbackOnRuntime
                 validator.validateOperation(op);
             }
 
+            if (!bootSuccess && error == null) {
```

Comment:
I suspect the error would be always null at this point, isn't it?

I can only see a modification of its value after `start` / `waitForSetup` and at the end of this method.
The `start` method invokes the `boot`, and the `waitForSetup` is only invoked after the `latch.countDown()`, which only happens after the boot method is finished, what I am missing for getting a not null value for this `error` instance variable?

It is also read by `getBootError()`, but it seems nobody that reads it writes it.